docs/business.yaml

```diff
@@ -239,6 +323,11 @@ paths:
validatorValue:[conversion],jsonPath:$.filing,message:'correction' is a required property,validator:required,validatorValue:[correction],jsonPath:$.filing,message:'courtOrder' is a required property,validator:required,validatorValue:[courtOrder],jsonPath:$.filing,message:'dissolution' is a required property,validator:required,validatorValue:[dissolution],jsonPath:$.filing,message:'incorporationApplication' is a required property,validator:required,validatorValue:[incorporationApplication],jsonPath:$.filing,message:'putBackOn' is a required property,validator:required,validatorValue:[putBackOn],jsonPath:$.filing,message:'registrarsNotation' is a required property,validator:required,validatorValue:[registrarsNotation],jsonPath:$.filing,message:'registrarsOrder' is a required property,validator:required,validatorValue:[registrarsOrder],jsonPath:$.filing,message:'registration' is a required property,validator:required,validatorValue:[registration],jsonPath:$.filing,message:'restoration' is a required property,validator:required,validatorValue:[restoration],jsonPath:$.filing,message:'specialResolution' is a required property,validator:required,validatorValue:[specialResolution],jsonPath:$.filing,message:'transition' is a required property,validator:required,validatorValue:[transition],jsonPath:$.filing,message:'unmanaged' is a required property,validator:required,validatorValue:[unmanaged]],error:'header': 'name': 'changeOfAddress', 'certifiedBy': 'full name', 'email': 'no_one@never.get', 'date': '2024-07-23', 'business': 'foundingDate': '2024-07-16T21:20:55.000+00:00', 'identifier': 'BC0883110', 'legalName': '0883110 B.C. LTD.', 'legalType': 'BC', 'changeOfAddress': 'legalType': 'BC', 'offices': 'registeredOffice': 'deliveryAddress': 'actions': ['addressChanged'], 'addressCity': 'RICHMOND', 'addressCountry': 'CA', 'addressRegion': 'BC', 'addressType': 'delivery', 'deliveryInstructions': '', 'postalCode': 'V8N 4R7', 'streetAddress': 'TEST D 1', 'streetAddressAdditional': '', 'mailingAddress': 'actions': ['addressChanged'], 'addressCity': 'RICHMOND', 'addressCountry': 'CA', 'addressRegion': 'BC', 'addressType': 'mailing', 'deliveryInstructions': '', 'postalCode': 'V8N 4R7', 'streetAddress': 'TEST M 1', 'streetAddressAdditional': '' is not valid under any of the given schemas,path:filing],filing:business:foundingDate:2024-07-16T21:20:55.000+00:00,identifier:BC0883110,legalName:0883110 B.C. LTD.,legalType:BC,changeOfAddress:legalType:BC,offices:registeredOffice:deliveryAddress:actions:[addressChanged],addressCity:RICHMOND,addressCountry:CA,addressRegion:BC,addressType:delivery,deliveryInstructions:,postalCode:V8N 4R7,streetAddress:TEST D 1,streetAddressAdditional:,mailingAddress:actions:[addressChanged],addressCity:RICHMOND,addressCountry:CA,addressRegion:BC,addressType:mailing,deliveryInstructions:,postalCode:V8N 4R7,streetAddress:TEST M 1,streetAddressAdditional:,header:certifiedBy:full name,date:2024-07-23,email:no_one@never.get,name:changeOfAddress "
```

Comment:
For this one, we have changed it to less 'scary' and shorter version in the latest update.  Consider changing it to
```yaml
rootCause: "errors:<comma delimited list of potential validation errors by json path>"
```